composables/useViewport.ts

```diff
@@ -1,7 +1,14 @@
 import { useWindowSize } from '@vueuse/core'
 
 export default function () {
-  const isMobile = computed(() => useWindowSize().width.value <= 768)
+  const { width } = useWindowSize()
```

Comment:
I already added this module https://www.npmjs.com/package/@nuxtjs/device. I think it's better to use that instead

Reply:
but this module checks the device type

using `isMobile` for example on a desktop with a window like this 
![CleanShot 2024-01-25 at 13 39 21@2x](https://github.com/kodadot/nft-gallery/assets/44554284/8a4e457b-df76-40b4-aaa1-14dd47d5fcc4)

returns `false` this is not what we want in this case

Reply:
i dont think common user have this size for a desktop